packages/infra/lib/api-stack.ts

```diff
@@ -706,6 +708,29 @@ export class APIStack extends Stack {
       apiAddress: apiLoadBalancerAddress,
     });
 
+    //-------------------------------------------
+    // Backups
+    //-------------------------------------------
+    if (this.isProd(props)) {
+      // create a daily backup plan for the API DB
+      new Backup(this, "APIDBBackup", {
+        backupPlanName: "APIDBBackupPlan",
+        resources: [BackupResource.fromRdsDatabaseCluster(dbCluster)],
+      });
+      // create a daily backup plan for the Medical Docs S3 bucket
+      new Backup(this, "APIMedicalDocsBucketBackup", {
+        backupPlanName: "APIMedicalDocsBucketBackupPlan",
+        resources: [BackupResource.fromArn(medicalDocumentsBucket.bucketArn)],
+      });
+    }
+    if (isSandbox(props.config) && sandboxSeedDataBucket) {
+      // create a daily backup plan for the Sandbox Seed Data S3 bucket
+      new Backup(this, "APISandboxSeedDataBucketBackup", {
+        backupPlanName: "APISandboxSeedDataBucketBackupPlan",
+        resources: [BackupResource.fromArn(sandboxSeedDataBucket.bucketArn)],
+      });
```

Comment:
moved to weekly - agreed this will rarely be changing, original thought was if we do change it (since we do need better sandbox data), it will be a week before we get a backup in case we need to do a restore, but this is NBD